### PR TITLE
#154: Add Task Copy() implementation of MsBoxStandardView class. Fix a bug on MsBoxCustomView

### LINKED ISSUE #154: Bug - Copy text from dialog causes exception
I have a dialog with text on it. When I select and copy the text using Ctrl+C, an exception arises and the program crashes. For some reason, the exception does not happen if the text is copied using mouse right-click and then copy.

Dialog built and shown like this:

```cs
private void ShowHelpDialog()
{
    Bitmap bitmap = new(AssetLoader.Open(new("avares://Pororoca.Desktop/Assets/Images/pororoca.png")));

    var msgbox = MessageBoxManager.GetMessageBoxStandard(
        new MessageBoxStandardParams()
        {
            ContentTitle = "title",
            ContentMessage = "content",
            WindowStartupLocation = WindowStartupLocation.CenterScreen,
            WindowIcon = new(bitmap),
            ButtonDefinitions = ButtonEnum.Ok
        });
    Dispatcher.UIThread.Post(async () => await msgbox.ShowAsync());
}
```

Exception in console:

```
Unhandled exception. System.NotImplementedException: The method or operation is not implemented.
   at MsBox.Avalonia.Controls.MsBoxStandardView.Copy()
   at CompiledAvaloniaXaml.XamlIlTrampolines.MsBox.Avalonia:MsBox.Avalonia.ViewModels.AbstractMsBoxViewModel+Copy_0!CommandExecuteTrampoline(Object, Object)
   at Avalonia.Markup.Xaml.MarkupExtensions.CompiledBindings.CommandAccessorPlugin.CommandAccessor.Command.Execute(Object parameter)
   at Avalonia.Input.KeyboardDevice.ProcessRawEvent(RawInputEventArgs e)
   at Avalonia.Input.InputManager.ProcessInput(RawInputEventArgs e)
   at Avalonia.Controls.TopLevel.HandleInput(RawInputEventArgs e)
   at Avalonia.X11.X11Window.DispatchInput(RawInputEventArgs args)
   at Avalonia.RawEventGrouper.Dispatch(RawInputEventArgs ev)
   at Avalonia.X11.X11PlatformThreading.RunLoop(CancellationToken cancellationToken)
   at Avalonia.Threading.DispatcherFrame.Run(IControlledDispatcherImpl impl)
   at Avalonia.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at Avalonia.Threading.Dispatcher.MainLoop(CancellationToken cancellationToken)
   at Avalonia.Controls.ApplicationLifetimes.ClassicDesktopStyleApplicationLifetime.Start(String[] args)
   at Pororoca.Desktop.Program.Main(String[] args) in /home/runner/work/Pororoca/Pororoca/src/Pororoca.Desktop/Program.cs:line 17
```

### PR DESCRIPTION
**Pull request by ashotjanibekyan**
Fixes #154

### CODE REVIEW

File: MsBox.Avalonia/Controls/MsBoxCustomView.axaml.cs [hunk 1]
```diff
@@ -29,8 +29,11 @@ public string GetButtonResult()
     public Task Copy()
     {
         var clipboard = TopLevel.GetTopLevel(this).Clipboard;
-        var vm = this.DataContext as MsBoxCustomViewModel;
```
Comment (ashotjanibekyan) [inline MsBox.Avalonia/Controls/MsBoxCustomView.axaml.cs:32]: this was always copying the entire content, not the selected part. Now we are only copying the entire text if nothing is selected

**Comment by GallaFrancesco** [comment]
What's the state of this? I'm just a user but throwing when Copy gets called can cause whole Avalonia applications to crash.

**Comment by ashotjanibekyan** [comment]
> What's the state of this? I'm just a user but throwing when Copy gets called can cause whole Avalonia applications to crash.

@GallaFrancesco the repo hasn't been updated for a while. You either have to wait for them to merge the PR or build the library yourself and use the dll directly, instead of Nuget (as a temp solution).

**Comment by GallaFrancesco** [comment]
I realized that `Copy` is implemented in `MsgBoxCustomView`, so it's possible to work around this provided that users have to configure the Custom properties.

**Pull request merged**